app/views/events/_edit.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="modal fade" id="donationeditModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+  <div class="modal-dialog modal-sm" role="document">
+    <div class="modal-content">
+      <div class="modal-body">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+
+        <% @event.donations.each do |event| %>
```

Comment:
This opens a modal that has a form for every single donation. It should only have a form for the donation we clicked. 😀